test/CMakeLists.txt

```diff
@@ -93,8 +95,10 @@ add_custom_target(valgrind_tests
   # The death tests will print their SIGABRT stacktrace under Valgrind. I have
   # not found a way to disable it.
   COMMAND ${VALGRIND_COMMAND} ./test_qsbr;
+  COMMAND ${VALGRIND_COMMAND} ./test_key_encode_decode;
   COMMAND ${VALGRIND_COMMAND} ./test_art;
+  #COMMAND ${VALGRIND_COMMAND} ./test_art_span;
```

Comment:
No commented out code

Reply:
I am planning to leave this stuff in for the agreed on quick merge and re-branch.  I don't want to have the figure out the 100s of little bits that I cut out only to have to get them all back into position.